rdbms/src/main/java/org/spine3/server/storage/jdbc/aggregate/query/Constants.java

```diff
@@ -28,36 +28,24 @@
 @SuppressWarnings("UtilityClass")
 /* package */ final class Constants {
```

Comment:
Pls define inner static classes `AggregateTable` and `EventCountTable` and move corresponding constants. And have two id cols. And, of course, do this for other storages.